lib/ArrayInterfaceCore/src/ArrayInterfaceCore.jl

```diff
@@ -616,6 +616,15 @@ Base.@propagate_inbounds function Base.getindex(ind::TridiagonalIndex, i::Int)
     end
 end
 
+"""
+    Label(label)
+
+A type that clearly communicates that `label` refers to a key-index mapping.
```

Comment:
This sounds like a very generic name, especially to clearly communicates that `label` refers to a key-index mapping". ;-) Maybe `IndexLabel`?

Reply:
agreed